### PR TITLE
changed FormikErrors and FormikTouched types to use same keys as Values

### PR DESCRIPTION
**Pull request by grzegorzmoskal**
Since one can find in docs `Form validation errors. Should match the shape of your form's values defined in initialValues` it's nice if we expres that in type definition.

### CODE REVIEW

**Comment by prichodko** [comment]
Hey @grzegorzmoskal, Formik already uses lockfile from Yarn. I don't think your PR should include another one.

**Comment by jaredpalmer** [comment]
Can u make changes so I can merge?

**Comment by grzegorzmoskal** [comment]
If you mean removing that `packages-lock` then I did 6 days ago ;)

File: src/formik.tsx [hunk 1]
```diff
@@ -21,16 +21,12 @@ export interface FormikValues {
 /**
  * An object containing error messages whose keys correspond to FormikValues.
  */
-export type FormikErrors = {
-  [field: string]: string;
-};
+export type FormikErrors<TValues> = { [field in keyof TValues]: string };
 
 /**
  * An object containing touched state of the form whose keys correspond to FormikValues.
```
[the remark this answers is not in the document]
Comment (jaredpalmer) [reply src/formik.tsx]: I forgot to hit "submit" by accident, i reviewed this like 5 days ago my b

**Pull request merged**